stc_game/Assets/Scripts/Targets/DestinationController.cs

```diff
@@ -1,37 +1,32 @@
 ﻿using System;
-using System.Collections;
-using System.Collections.Generic;
 using UnityEngine;
 using UnityEngine.EventSystems;
 
 public class DestinationController : MonoBehaviour
 {
     public Camera cam;
-    public GameObject hero;
+    public Hero hero;
     public GameObject target;
     public Vector3 floorOffset;
-
-    
+    private StateController controller;
 
     // Start is called before the first frame update
     void Start()
     {
-        hero = GameObject.FindGameObjectWithTag("Player");
+        hero = GameObject.FindGameObjectWithTag("Player").GetComponent<Hero>();

```

Comment:
again, we'll get this into a service, just note for now all the files we'll have to touch to refactor for the hero data

Reply:
ok